src/Nordea/Components/NumberInput.elm

```diff
@@ -156,6 +156,7 @@ getAttributes config =
         , config.placeholder |> Maybe.map placeholder
         , config.onInput |> Maybe.map onInput
         , config.onBlur |> Maybe.map onBlur
+        , Just config.isDisabled |> Maybe.map Attributes.disabled
```

Comment:
Instead, I suggest to remove it from the config. It is already supported to set disabled as a normal attribute:

`|> NumberInput.view [ disabled True ]`

This is also the interface we use for disabled for the other components as well

```suggestion
```

Reply:
@fredrikloberg Good point! Do you think we should then remove the exposed `withIsDisabled` ? 🤔If that is not the right way to do disabling

```
withIsDisabled : Bool -> NumberInput msg -> NumberInput msg
withIsDisabled val (NumberInput config) =
    NumberInput { config | isDisabled = val }
```

Reply:
Yea 👍

Reply:
@fredrikloberg Great 😍 and done! Could you please take one more look at it ?